sample_parameter_files/turbulence_recommended_params

```diff
@@ -0,0 +1,131 @@
+
+
+####### Restart parameters #######
+ntout_restart		=	-1		# restart output frequency in timesteps
+dtout_restart		=	0.005d0		# restart output frequency in time
+do_restart 		= 	0
+restart_file_name	=	""
+read_v_turb             =       0               # 1 if you are restarting from a simulation that includes turbulence, 0 otherwise
+force_restart_dump	=	0		# keep this 0 to maintain output frequency, 1 forces output at start
+
+########## M1 settings ##########
+do_M1		 	=       1			#1 for M1 transport scheme
+v_order			=	-1
+evolution_radii		=	6.0d7			#radii to evolve neutrinos out to
+extraction_radii	=	5.0d7			#radii to extract neutrinos from
+number_species		=	3			#number of neutrino species, limited 
+							#options will work, must be comensurate with table
+number_groups		=	18			#number of energy groups, limited 
+							#options will work, must be comensurate with table
+opacity_table 		=	"NuLib_LS180_noweak_rho82_temp65_ye51_ng18_ns3_Itemp65_Ieta61_version1.0_20141111.h5"
+number_eas		=	3			#number of opacity variables to read in
+M1closure		=	'ME'
+testcase		=	0			#if you want a test case set this to the
+							#test case number, along with M1test as "initial_data"
+include_epannihil_kernels =	0			#for full nux thermal treatment
+include_nes_kernels	= 	1			#to read in inelastic neutrino electron scattering kernels
+nes_evolution_type	=	1			#0: no nes, 1: explicit nes, 2: implicit nes
+energycoupling_evolution_type =	1			#0: no coupling, 1: explicit coupling, 2: implicit coupling
+
+
+M1_control		=	1			#1 means you must specify the settings for each phase
+							# in the parameter file, collapse, bounce and postbounce.	
+
+M1_phase1phase2_density =	1.0d12			#central density to switch to phase 2
+M1_phase2phase3_pbtime  =	0.02d0			#post bounce time to switch to phase3
+
+M1_phase1_reconstruction =	'tvd'			#reconstruction method for both matter and neutrinos (phase1 must match above)
+M1_phase2_reconstruction =	'ppm'
+M1_phase3_reconstruction =	'ppm'
+
+M1_phase1_cffac 	=	0.5d0			#CFL factor, typically needs to be reduced near bounce
+M1_phase2_cffac 	=	0.2d0			
+M1_phase3_cffac		=	0.5d0
+
+M1_phase1_ns		=	1			#number of species to evolve
+M1_phase2_ns		=	3
+M1_phase3_ns		=	3
+
+M1_phase1_ies_way	=	1			#0 is none, 1 is explicit inelastic scattering, 2 is implicit (phase1 must match above)
+M1_phase2_ies_way	=	2			
+M1_phase3_ies_way	=	1
+
+M1_phase1_encpl_way	=	1			#0 is none, 1 is explicit energycoupling, 2 is implicit (phase1 must match above)
+M1_phase2_encpl_way	=	1			
+M1_phase3_encpl_way	=	1
+
+do_M1_extra_heating	=	0
+M1_heat_fac		=	1.0d0
+
+####### Neutrino parameters #######
+fake_neutrinos		=	0		# 1 for ANY fake neutrino scheme
+
+####### Atmosphere parameters #######
+atmo_rho_rel_min	=	0.0d0
+atmo_rho_abs_min	=	1.5d3
+atmo_fac		=	0.1d0
+
+####### Rotation #######
+do_rotation		=	0	# 1 for 1D rotation
+
+###### Turbulence #######
+do_turbulence           =       1       # 1 for turbulence of Couch et al. (2019)
```

Comment:
Can you update the other recommend parameters file to have this line (but as a 0)